diff --git a/moveit_setup_assistant/CMakeLists.txt b/moveit_setup_assistant/CMakeLists.txt
index e06c9011c..e0b7e6962 100644
--- a/moveit_setup_assistant/CMakeLists.txt
+++ b/moveit_setup_assistant/CMakeLists.txt
@@ -46,9 +46,12 @@ pkg_check_modules(YAMLCPP REQUIRED yaml-cpp>=0.5)
 include_directories(${YAMLCPP_INCLUDE_DIRS})
 link_directories(${YAMLCPP_LIBRARY_DIRS})
 
+pkg_check_modules(OGRE_OV OGRE)
+
 catkin_package(
   INCLUDE_DIRS
     include
+    ${OGRE_OV_INCLUDE_DIRS}
   LIBRARIES
     ${PROJECT_NAME}_tools
   CATKIN_DEPENDS
@@ -153,7 +156,7 @@ install(DIRECTORY templates DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
 # Unit tests
 if(CATKIN_ENABLE_TESTING)
   find_package(moveit_resources REQUIRED)
-  include_directories(${moveit_resources_INCLUDE_DIRS})
+  include_directories(${moveit_resources_INCLUDE_DIRS} ${OGRE_OV_INCLUDE_DIRS})
   catkin_add_gtest(test_reading_writing_config test/moveit_config_data_test.cpp)
   target_link_libraries(test_reading_writing_config  ${PROJECT_NAME}_tools
   ${catkin_LIBRARIES}
